src/webui/abstractwebapplication.cpp

```diff
@@ -386,3 +374,22 @@ const QStringMap AbstractWebApplication::CONTENT_TYPE_BY_EXT = {
     { "png", Http::CONTENT_TYPE_PNG },
     { "js", Http::CONTENT_TYPE_JS }
 };
+
+QStringMap AbstractWebApplication::parseCookie(const Http::Request &request) const
+{
+    // [rfc6265] 4.2.1. Syntax
+    QStringMap ret;
+    const QVector<QStringRef> cookies = request.headers.value(QLatin1String("cookie"))
+                                            .splitRef(';', QString::SkipEmptyParts);
+    for (const auto &cookie : cookies) {
+        const int idx = cookie.indexOf('=');
+        if (idx < 0)
+            continue;
+
+        const QString name = cookie.left(idx).trimmed().toString();
```

Comment:
Because `QStringMap::value()` returns `const QString` and this temporary object gets out of scope at the very next line.